rfcs/pullpanda_analytics.md

```diff
@@ -0,0 +1,65 @@
+# RFC 35: Add third-party GitHub App: Pull Panda
+
+## Summary
+
+At TPAC 2019 we set out some [priorities for 2020](https://docs.google.com/document/d/1gie7LFb6cAUfabY86MYuWM7m7ux_FaKhDkLdpz0zWkg/edit),
+which include increasing PR review velocity and reduce the number of stale PRs.
+
+In order to track progress over time,
+install the [Pull Panda](https://pullpanda.com/) GitHub App
+for the web-platform-tests/wpt repository (or all repos in the org).
```

Comment:
We should decide whether to enable it for only wpt or the entire org.

Reply:
If it doesn't create statuses, deployments or anything like that, then enabling it for the whole org seems fine.